services/core-api/app/api/securities/resources/bond.py

```diff
@@ -26,7 +26,7 @@ def get(self):
         mine = Mine.find_by_mine_guid(mine_guid)
 
         if mine is None:
-            raise NotFound('No mine found with the provided mine_guid.')
+            raise NotFound('No mine was found with the guid provided.')
```

Comment:
platonically, this should just return `[]`. it's a querystring param.

Reply:
`raise NotFound()`?

Reply:
Nevermind, I see what you mean now... Are you sure that would be good behaviour? The user is trying to get all bonds on a specific mine, if they provide an invalid mine GUID, I think we should raise not found. I think returning an empty [] for bonds is only appropriate if they search for a valid mine that happens to have no bonds.

Reply:
querystring param should be treated like a `WHERE` clause, we are enforcing that mine_guid is provided because we refuse to return all securities across all mines. 

ivory tower stuff, not the end of the world.

Reply:
I understand more what you mean now. I'm happy to change it to return an empty result set then.